<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>combat_tracker_mainClass</class>
 <widget class="QMainWindow" name="combat_tracker_mainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>combat_tracker_main</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="contentArea" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Current Round</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="current_round_integer_label">
           <property name="text">
            <string>PLACEHOLDER</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="startCombatButton">
           <property name="text">
            <string>Start combat</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="addParticipant">
           <property name="text">
            <string>Add a participant</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="sortParticipantsButton">
           <property name="text">
            <string>Sort participants</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="quitButton">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_participants"/>
    <addaction name="actionLoad_participants"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave_participants">
   <property name="text">
    <string>Save participants</string>
   </property>
  </action>
  <action name="actionLoad_participants">
   <property name="text">
    <string>Load participants</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="combat_tracker_main.qrc"/>
 </resources>
 <connections/>
</ui>
